src/NextGen/DonationForm/FormDesigns/DefaultFormDesign/js/layouts/NodeWrapper.tsx

```diff
@@ -0,0 +1,17 @@
+import type {Element} from '@givewp/forms/types';
+import classNames from 'classnames';
+import type {NodeWrapperProps} from '@givewp/forms/propTypes';
+
+export function NodeWrapper({nodeType, type, name, htmlTag: Element = 'div', children}: NodeWrapperProps) {
+    return (
+        <Element
+            className={classNames({
+                [`givewp-${nodeType}`]: nodeType,
+                [`givewp-${nodeType}-${type}`]: type,
+                [`givewp-${nodeType}-${type}-${name}`]: name,
```

Comment:
Isn't `name` optional?

Reply:
yes, this is a feature of the classNames library where it only adds the className if the argument is not null.